scripts/path_plans.py

```diff
@@ -260,80 +150,217 @@ def _rotationMatrix_orient_z_axis(self, local_point):
         # Rotation Matrix
         # https://en.wikipedia.org/wiki/Rotation_formalisms_in_three_dimensions#Formalism_alternatives
         rot_matrix = np.stack((vector_x, vector_y, vector_z), axis=1)
-        
+
         return rot_matrix
 
 
+class NewInclinedPlane(DetectedObject):
+    """A container for the incline plane path plan"""
+    def __init__(self, size, locator, orientation, count=(5, 5), clearance=0.25, plane_scale=(2, 2),\
+                 slope=0.5, offset=0.25):
+        # type: (List[float], List[float], np.ndarray, Tuple[int,int], float, Tuple[float,float], float, float) -> None
+        """Initialization function
+
+        @param size:        list[x,y,z] dimensions of presented part
+        @param locator:     center of presented part in world frame coordinates
+        @param orientation: rotation of presented part in world frame coordinates
+        @param count:       Tuple (X,Z) with the count of points along each direction of the plane
+        @param clearance:   Distance between each plane and the part before the planes are inclined
+        @param plane_scale: Tuple (X,Z) describing the how the planes dimensions are scaled relative to the part's
+        @param slope:       Multiplier from 0 (no tilt) to 1 (maximum tilt) to adjust the incline of each plane
```

Comment:
Scalar Multiplier makes sense. What is maximum tilt? And does that change given different sized objects?

Reply:
Maximum tilt means the plane is coincident with the bounding box. Will clarify if the docstring.